client/src/pages/Messages.tsx

```diff
@@ -277,13 +279,43 @@ const Profile = () => {
   const [messages, setMessages] = useState<WebSocketMessage[]>([])
   const { sendMessage, lastMessage } = useContext(WebSocketContext)
   const { user } = useUser()
+  const { session } = useSession()
+  const groupId = '1980b837-6506-4b53-8363-f8d21c43822e'
 
   useEffect(() => {
     if (lastMessage) {
       setMessages((msgs) => [...msgs, lastMessage])
     }
   }, [lastMessage])
 
+  useEffect(() => {
+    const getPastMessages = async () => {
+      const token = await session?.getToken()
+
+      if (token) {
+        const result = await getMessageForGroup(groupId, token)
+        const mapped: WebSocketMessage[] = result.map((dbMessages) => ({
+          action: '@@MESSAGE/RECEIVE',
+          payload: {
+            message: dbMessages.message,
+            receiverId: dbMessages.group_id,
+          },
+          metadata: {
+            authorId: dbMessages.author_id,
+            receivedAt: dbMessages.created_at,
+          },
+          id: dbMessages._id,
+        }))
```

Comment:
we have to map the Database messages into the Websocket messages. We should probably write down somewhere the shapes we expect